python/pyspark/sql/dataframe.py

```diff
@@ -1311,6 +1329,28 @@ def fill(self, value, subset=None):
     fill.__doc__ = DataFrame.fillna.__doc__
 
 
+class DataFrameStatFunctions(object):
+    """Functionality for statistic functions with :class:`DataFrame`.
+    """
+
+    def __init__(self, df):
+        self.df = df
+
+    def cov(self, col1, col2):
```

Comment:
take a look at how we do it for DataFrameNaFunctions so we don't need to duplicate the doc.